### PR TITLE
fix: node fs instrumentation

### PR DESCRIPTION
**Pull request by neo773**
This PR adds an options to enable node fs instrumentation. default value is false

/claim #6553

### CODE REVIEW

**Review by Vadman97 — COMMENTED** [review]
great! testing locally now.
could you also bump the version of the [package.json](https://github.com/highlight/highlight/blob/7d3c31f28f1baba0206b519dcea31b3edc4cfa77/sdk/highlight-node/package.json#L3) to 3.3.0 and add a minor release note to the [changelog](https://github.com/highlight/highlight/blob/7d3c31f28f1baba0206b519dcea31b3edc4cfa77/sdk/highlight-node/CHANGELOG.md)?

File: sdk/highlight-node/src/types.ts [hunk 1]
```diff
@@ -21,4 +21,10 @@ export interface NodeOptions extends HighlightOptions {
 	 * This app's version ideally set to the latest deployed git SHA.
 	 */
 	serviceVersion?: string
+	/**
+	 * Enables node fs instrumentation @default false
+	 * see .
+	 * {@link https://opentelemetry.io/docs/instrumentation/js/libraries/#registration}
+	 */
+	enableFsInstrument?: boolean
```
Comment (Vadman97) [inline sdk/highlight-node/src/types.ts:29]: can we call this option `enableFsInstrumentation`?

**Review by Vadman97 — APPROVED** [review]
tested locally, with the flag `true`, fs is instrumented, without the flag, we no longer get fs spans. thanks for working on this!

**Comment by neo773** [comment]
@Vadman97 About bumping the package version should I bump major or minor? since it does change existing functionality

**Comment by Vadman97** [comment]
you're right, technically this is changing functionality of the sdk, but our backend has been dropping fs traces for some time now so this wouldn't change the experience in highlight. i'd stick to a minor version bump for that reason.

**Comment by neo773** [comment]
@Vadman97 Pushed!

**Comment by Vadman97** [comment]
great, could you also update the [CHANGELOG.md](https://github.com/highlight/highlight/blob/7d3c31f28f1baba0206b519dcea31b3edc4cfa77/sdk/highlight-node/CHANGELOG.md)

**Comment by neo773** [comment]
@Vadman97 Done, sorry totally slipped my mind.

File: sdk/highlight-node/src/client.ts [hunk 2]
```diff
@@ -75,7 +75,13 @@ export class Highlight {
 			},
 			spanProcessor: this.processor,
 			traceExporter: exporter,
-			instrumentations: [getNodeAutoInstrumentations()],
+			instrumentations: [
+				getNodeAutoInstrumentations({
+					'@opentelemetry/instrumentation-fs': {
+						enabled: options.enableFsInstrument ?? false,
```
Comment (Vadman97) [inline sdk/highlight-node/src/client.ts:81]: should be `enableFsInstrumentation`

**Review by Vadman97 — APPROVED** [review]

**Comment by Vadman97** [comment]
sorry one last thing @neo773 , could you do a minor version bump of [@highlight-run/apollo](https://github.com/highlight/highlight/blob/24b27bf02aafafc2c6ce37bbcfe0edcd43049cb8/sdk/highlight-apollo/package.json#L3), [@highlight-run/nest](https://github.com/highlight/highlight/blob/24b27bf02aafafc2c6ce37bbcfe0edcd43049cb8/sdk/highlight-nest/package.json#L3), [@highlight-run/next](https://github.com/highlight/highlight/blob/24b27bf02aafafc2c6ce37bbcfe0edcd43049cb8/sdk/highlight-next/package.json#L3), and [@highlight-run/remix](https://github.com/highlight/highlight/blob/24b27bf02aafafc2c6ce37bbcfe0edcd43049cb8/sdk/highlight-remix/package.json#L3) as they all depend on @highlight-run/node?

**Comment by neo773** [comment]
Pushed!

**Pull request merged**